package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "dependencies": {
+    "@rails/webpacker": "^3.0.1",
+    "autoprefixer": "^7.1.3",
+    "babel-core": "^6.26.0",
+    "babel-loader": "7.x",
+    "babel-plugin-syntax-dynamic-import": "^6.18.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.6.0",
+    "coffee-loader": "^0.8.0",
+    "coffee-script": "^1.12.7",
+    "compression-webpack-plugin": "^1.0.0",
+    "css-loader": "^0.28.5",
+    "extract-text-webpack-plugin": "^3.0.0",
+    "file-loader": "^0.11.2",
+    "glob": "^7.1.2",
+    "js-yaml": "^3.9.1",
+    "node-sass": "^4.5.3",
+    "path-complete-extname": "^0.1.0",
+    "postcss-loader": "^2.0.6",
+    "postcss-smart-import": "^0.7.5",
+    "precss": "^2.0.0",
+    "rails-erb-loader": "^5.2.1",
+    "resolve-url-loader": "^2.1.0",
+    "sass-loader": "^6.0.6",
+    "style-loader": "^0.18.2",
+    "webpack": "^3.5.5",
+    "webpack-manifest-plugin": "^1.3.1",
+    "webpack-merge": "^4.1.0"
+  },
+  "devDependencies": {
+    "webpack-dev-server": "^2.7.1"
```

Comment:
We should add eslint, right @sbalay 

This is our conf
```
    "babel-eslint": "^7.2.3",
    "babel-plugin-transform-object-rest-spread": "^6.23.0",
    "eslint": "^3.19.0",
    "eslint-config-airbnb": "^15.0.1",
    "eslint-config-prettier": "^2.1.1",
    "eslint-plugin-flowtype": "^2.34.0",
    "eslint-plugin-import": "^2.3.0",
    "eslint-plugin-jsx-a11y": "^5.0.3",
    "eslint-plugin-prettier": "^2.1.1",
    "eslint-plugin-react": "^7.0.1",
    "prettier": "^1.4.0",
    "prettier-eslint": "^6.2.3",
```

Reply:
Yeah, @mtejedorwolox use the rules from here https://github.com/Wolox/wolmo-bootstrap-react-native/blob/master/generators/app/templates/.eslintrc.ejs

Reply:
@mtejedorwolox you will also need to add the babel plugins to the babel config.